javascript/code-challenges/challenges-03.test.js

```diff
@@ -0,0 +1,300 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function called addTwo that takes in an array and adds two to every value using a for loop. Place the new value in a new array. Return the new array.
+------------------------------------------------------------------------------------------------ */
+
+const addTwo = (arr) => {
+  // Solution code here...
+
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named typeNum that, given an array as input, uses filter to return an array containing only the numbers.
+
+For example, typeNum([1, 'bob' ,3]) returns [1,3].
+------------------------------------------------------------------------------------------------ */
+
+const typeNum = (arr) => {
+  // Solution code here...
+};
```

Comment:
- This one is a bit tricky but perhaps if you use [typeof](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof) for this one, it will help 👍 
- we will set up our filter just like you did for challenge below, and then do some sort of check there to determine what values are nums